inst/rmarkdown/templates/jedm_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,114 @@
+knitr::opts_chunk$set(echo=FALSE, cache=FALSE, cache.comments=FALSE, message=FALSE, warning=FALSE, tidy=TRUE)
+
+# Seed for random number generation:
+set.seed(7777)
+```
+
+# Paper format
+
+Manuscripts should be formatted for letter sized paper, one side only, leaving 2.75cm margins on the right and left sides and 2.25cm on the top and bottom. Body font should be Times Roman 12pt and sections, title, and authors font should be Helvetica.
+
+# Style files
+
+For LaTeX, a style file named \texttt{jedm.cls} is provided on the journal's web site.  An MS Word file containing this example text is also provided, \texttt{jedm.doc}, which can be used as a template (see the Styles menu). Note that copy/pasting into the template may not ensure correct formatting but selecting a region of text and applying the appropriate style should always work. 
+
+# Figures and tables
+
+Each figure and table must be mentioned in the text, and must be numbered consecutively in order of appearance (with captions in lower case). For the review process, the figures should be integrated into the text rather than being inserted at the end of the document.
+
+As JEDM is an electronic journal, authors are encouraged to use colours when it enhances visibility and understanding of figures.
+
+Examples of a figure and a table are given in Figure~\ref{tab:fig1} and Table~\ref{tab:1}.
+
+\begin{figure}
+  \centerline{\fbox{\parbox{.66\textwidth}{\centering \raisebox{0pt}[2cm][2cm]{%
+        \begin{tabular}{c}
+          \cellcolor{blue!35}{\sf This is a figure}\\
+          \cellcolor{blue!25}{\sf And yes, you can use colors!}\\
+          \cellcolor{blue!15}{\sf As long as it enhances visibility and understanding}\\
+        \end{tabular}
+  }}}}
+  \caption{This is the figure's caption.  It should be a centered paragraph of width 193mm (7.6in). Font size should be 11pt.}
+  \label{tab:fig1}
+\end{figure}
+
+\begin{table}
+  \caption{This is an example of a table that lists the margins of this template.  Captions should follow the same rules as a figure, except that they are put on top of the table.}\vspace*{1ex}
+  \label{tab:1}
+  \centering
+  \begin{tabular}{| l | l |}
+    \hline
+    \multicolumn{1}{|c|}{\textbf{Margin}} & \multicolumn{1}{c|}{\textbf{Size}} \\
+    \hline
+    left&2.75cm\\
+    right&2.75cm\\
+    top&2.25cm\\
+    bottom&2.25cm\\
+    \hline
+  \end{tabular}
+\end{table}
+
+# Appendices
+
+Supplementary technical material (e.g., mathematical proofs or descriptions of experimental procedures) should be collected in an appendix at the end of the paper (before the acknowledgements and the references sections).
+
+# Footnotes and acknowledgments
+
+Footnotes should be used sparingly and indicated by consecutive superscript numbers in the text. Material to be footnoted should appear at the bottom of the page on which it is referenced. Acknowledgments and grant numbers should be put into a separate `Acknowledgment' section right before the list of references.
```

Comment:
I think that is the wrong quote after Acknowledgment ? Am I right ?

Reply:
FYI, `` ` '`` is the LaTeX syntax for a pair of curly single quotes (similarly, ``` `` ''``` is curly double quotes). However, here we shouldn't use the LaTeX syntax, but just two straight single quotes `` ' '``.

Reply:
Oh thank you ! I did not know that.